jupyterhub/orm.py

```diff
@@ -1141,7 +1141,6 @@ def new(
         expires_in=None,
         client_id=None,
         oauth_client=None,
-        return_orm=False,
```

Comment:
This argument was only partially added by mistake, I think for internal testing. It doesn't do anything.